src/workloads/tpch/denormalized/1/Q1.yml

```diff
@@ -0,0 +1,20 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 1 against the denormalized schema for scale 1.
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+
+Actors:
+- Name: TPCHDenormalizedQuery1
+  Type: RunCommand
+  Database: tpch
+  Phases:
+  - LoadConfig:
+      Path: ../../../../phases/tpch/denormalized/Q1.yml
+      Key: TPCHDenormalizedQuery1
+      Parameters:
+        Query1Delta: -90
```

Comment:
There is no need to specify these parameters if they are the same as the default, as proven by the `Query1Delta` problem you pointed out in an earlier comment. However, I agree with you that it makes more sense to encode the parameters directly in the workload definition, though it might be a bit of duplication.

Possibly for scale 10, and/or possibly for future enhancements to the benchmark, we may want the ability to instrument parameters for these queries that are non-default (a true and complete implementationcof the benchmark would include randomized parameters and update functions).

For now this qualifies as overengineering, so if you feel strongly that I should remove the parameters from here and rely on the defaults instead I can do that.

Reply:
Oh I was just saying. I think we can keep as is. :)